### PR TITLE
Visitor can randomize movie

### PR DESCRIPTION
**Pull request by Cma-B**
### User Story
```
As a visitor
In order to choose a movie
I want to be able to randomise it
```
## Changes proposed in this pull request:
* Adds movie component and feature test 
* Adds two json files as mock data for test environment
## What I have learned working on this feature:
* randomize items in client side and send a request to api side

## Screenshots (Client)

image

### CODE REVIEW

File: cypress/fixtures/another_random_movie.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      12,
+      878
+    ],
+    "title": "Batman ",
+    "vote_average": 7.5,
+    "overview": "Batman bits up poor people.",
+    "release_date": "1999-05-15"
+  }
+}
+
```
Comment (Alex-Bellamy) [inline cypress/fixtures/another_random_movie.json]: perhaps remove this line?

File: cypress/fixtures/random_movie.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      12,
+      878
+    ],
+    "title": "Star Wars",
+    "vote_average": 8.2,
+    "overview": "Princess Leia is captured and held hostage by the evil Imperial forces in their effort to take over the galactic Empire. Venturesome Luke Skywalker and dashing captain Han Solo team together with the loveable robot duo R2-D2 and C-3PO to rescue the beautiful princess and restore peace and justice in the Empire.",
+    "release_date": "1977-05-25"
+  }
+}
+
```
Comment (Alex-Bellamy) [inline cypress/fixtures/random_movie.json]: another excess line?

**Review by Alex-Bellamy — COMMENTED** [review]
you may want to make a couple of minor amendments.

File: cypress/fixtures/another_random_movie.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "original_language": "en",
+    "original_title": "Batman",
+    "genre_ids": [
+      28,
+      12,
+      878
+    ],
+    "title": "Batman ",
+    "vote_average": 7.5,
+    "overview": "Batman bits up poor people.",
```
Comment (oliverochman) [inline cypress/fixtures/another_random_movie.json:20]: typo

File: cypress/integration/visitors_can_randomize_movies.feature.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      url: 'http://localhost:3000/api/v1/movies/random',
+      response: "fixture:another_random_movie.json"
+    });
+    cy.wait(1000)
+    cy.get("button").should("contain", "Randomize Movie").click();
+    cy.get('#random-movie').within(() => {
+      cy.get('#movie-title').should("contain", "Batman")
+      cy.get('#movie-overview').should("contain", "Batman bits up poor people.")
+      cy.get('#movie-release-date').should("contain", "1999-05-15")
+      cy.get('#movie-rating').should("contain", "1393 has rated this movie. It has an average ratings of 7.5")
```
Comment (oliverochman) [inline cypress/integration/visitors_can_randomize_movies.feature.js]: "1393 has rated this movie."

restructure this sentence, sound a bit weird

**Review by oliverochman — CHANGES_REQUESTED** [review]

**Review by viktorsommar — APPROVED** [review]

**Review by Felix-Willborg — APPROVED** [review]
good job guys.

**Pull request merged**